.scripts/regression-compare.yaml

```
@@ -64,7 +64,7 @@ languages:
     oldArgs:
       - --v3
       - --use:@autorest/python@5.4.0
```

Looks like this is `5.5.0` now, we can bump this now too

```suggestion
      - --use:@autorest/python@5.5.0
```